File: ds/set.py
# https://atcoder.jp/contests/abc217/tasks/abc217_b

s = set()
s.add('ABC')
s.add('ARC')
s.add('AGC')
s.add('AHC')
for i in range(3):
    s.remove(input())
print(s.pop())

File: ds/set.cc
// https://atcoder.jp/contests/abc217/tasks/abc217_b

#include <bits/stdc++.h>
using namespace std;

int main() {
  set<string> s;
  s.insert("ABC");
  s.insert("ARC");
  s.insert("AGC");
  s.insert("AHC");
  for (int i = 0; i < 3; i++) {
    string S;
    cin >> S;
    s.erase(S);
  }
  cout << *s.begin() << endl;
  return 0;
}
